src/renderer/components/notification-comp.tsx

```diff
@@ -74,50 +100,124 @@ export default class NotificationComp extends React.Component<{}, IState> {
-                            <path d='M28 17.1029V13.4094C28 12.6528 27.56 11.9425 26.8467 11.5534C25.7833 10.9728 23.48 10 20 10C16.52 10 14.2167 10.9728 13.1533 11.5565C12.44 11.9425 12 12.6528 12 13.4094V18.9559L24.6667 22.3529V24.8235C24.6667 25.1571 24.44 25.3918 24.0533 25.5678L20 27.4485L15.9233 25.5585C15.56 25.3918 15.3333 25.1571 15.3333 24.8235V22.9706L12 22.0441V24.8235C12 26.3491 12.9433 27.6462 14.4433 28.3225L20 31L25.5333 28.3349C27.0567 27.6462 28 26.3491 28 24.8235V20.1912L15.3333 16.7941V13.9746C16.24 13.57 17.78 13.0882 20 13.0882C22.22 13.0882 23.76 13.57 24.6667 13.9746V16.1765L28 17.1029Z' fill='#0098FF' />
-                            <path d='M28 17.1029V13.4094C28 12.6528 27.56 11.9425 26.8467 11.5534C25.7833 10.9728 23.48 10 20 10C16.52 10 14.2167 10.9728 13.1533 11.5565C12.44 11.9425 12 12.6528 12 13.4094V18.9559L24.6667 22.3529V24.8235C24.6667 25.1571 24.44 25.3918 24.0533 25.5678L20 27.4485L15.9233 25.5585C15.56 25.3918 15.3333 25.1571 15.3333 24.8235V22.9706L12 22.0441V24.8235C12 26.3491 12.9433 27.6462 14.4433 28.3225L20 31L25.5333 28.3349C27.0567 27.6462 28 26.3491 28 24.8235V20.1912L15.3333 16.7941V13.9746C16.24 13.57 17.78 13.0882 20 13.0882C22.22 13.0882 23.76 13.57 24.6667 13.9746V16.1765L28 17.1029Z' fill='url(#paint1_radial)' />
-                            <defs>
-                                <linearGradient id='paint0_linear' x1='20' y1='0' x2='20' y2='40' gradientUnits='userSpaceOnUse'>
-                                    <stop stopColor='white' stopOpacity='0.2' />
-                                    <stop offset='1' stopColor='white' stopOpacity='0' />
-                                </linearGradient>
-                                <radialGradient id='paint1_radial' cx='0' cy='0' r='1' gradientUnits='userSpaceOnUse' gradientTransform='translate(20.0278 10) rotate(90) scale(14.2187 20.1481)'>
-                                    <stop stopColor='white' stopOpacity='0.4' />
-                                    <stop offset='1' stopColor='white' stopOpacity='0' />
-                                </radialGradient>
-                            </defs>
-                        </svg>
+            <div className={containerClass} style={{ height: containerHeight }}>
+                <div
+                    className='main-container'
+                    role='alert'
+                    style={bgColor}
+                    onContextMenu={this.eventHandlers.onContextMenu}
+                    onClick={this.eventHandlers.onClick(id)}
+                    onMouseEnter={this.eventHandlers.onMouseEnter(id)}
+                    onMouseLeave={this.eventHandlers.onMouseLeave(id)}
+                >
+                    <div className='logo-container'>
+                        <div className='logo'>
+                            <svg width='40' height='40' viewBox='0 0 40 40' fill='none' xmlns='http://www.w3.org/2000/svg'>
```

Comment:
I would prefer to have the SVG as an asset instead of an inline svg, if it is a static svg.